web/src/features/Dashboard/components/DashboardVersionCard.tsx

```diff
@@ -520,13 +521,19 @@ const DashboardVersionCard = (props: Props) => {
                     ""
                   )}
                   <p
-                    className={`checks-running-text u-fontSize--small u-lineHeight--normal u-fontWeight--medium ${
+                    className={`checks-running-text u-fontSize--small u-lineHeight--normal u-fontWeight--medium 
+                    } ${
                       preflightState.preflightsFailed
                         ? "err"
                         : preflightState.preflightState === "warn"
                         ? "warning"
                         : ""
-                    }`}
+                    }
+                     ${
+                       !selectedApp && currentVersion?.status === "deploying"
+                         ? "without-btns"
```

Comment:
if no selectedApp and currentVersion is deploying, the two buttons will not be rendered. Therefore, the paragraph needs to transform a bit more.